csatar-plugins/forms/lang/en/lang.php

```diff
@@ -41,18 +42,18 @@
                     'actionDeleteKeywordDescr'  => 'Specify a keyword for deleting record. The default keyword is "delete", for example "/teams/123/delete" will team with id 123.',
                 ],
                 'propertiesValidation' => [
-                    'formNotSelected' => "Please select a form from the dropdown list. If list is empty, you need to create a form in From menu.",
+                    'formNotSelected'       => "Please select a form from the dropdown list. If list is empty, you need to create a form in From menu.",
```

Comment:
Szerintem ez a hibaüzi nem a leghelyesebb, mert a menüt pl "Forms"-nak hívják. S szerintem vmi ilyesmi jobb lenne: "Please select a form from the dropdown list. If the form you're looking for is not there, you need to create it in the Forms menu."
S ezenkívül ez a hibaüzi leblokkolja a felhasználót: ha egy form sincs a dropdownban, s nem tud semmit kiválasztani, akkor igazából nem tud a "Forms" menüre navigálni, mert se a hibaüzit, se az oldalt nem lehet bezárni. Na de gondolom, az nem fog előfordulni, hogy üres legyen, mert már most van elég sok form ott, igaz?

Reply:
A hibaüzenetet átírtam.
A blokkolással nem tudom mit kezdjek, ugyanígy működik a Builder Plugin Record list komponsensénél is. A validálást kivenni nem akarom, mert akkor a frontend romlik el ha nincs form kiválasztva. Egy refresh amúgy megoldja.